tests/testthat/test-http-header-removal.R

```diff
@@ -0,0 +1,83 @@
+context("test-http-headers")
+
+test_that("hop-by-hop headers removed from Rook requests", {
+  rook_req <-
+    list(
+      HTTP_CONNECTION = "keep-alive",
+      HTTP_UPGRADE_INSECURE_REQUESTS = "1",
+      HTTP_ACCEPT = "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8",
+      HTTP_ACCEPT_LANGUAGE = "en-US,en;q=0.9",
+      QUERY_STRING = "",
+      httpuv.version = structure(list(c(1L, 4L, 5L, 9000L)), class = c("package_version",
+                                                                       "numeric_version")),
+      SERVER_NAME = "127.0.0.1",
+      SCRIPT_NAME = "",
+      SERVER_PORT = "8600",
+      REMOTE_PORT = "37514",
+      PATH_INFO = "/",
+      rook.version = "1.1-0",
+      REMOTE_ADDR = "127.0.0.1",
+      HEADERS = c(
+        accept = "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8",
+        `accept-encoding` = "gzip, deflate, br",
+        `accept-language` = "en-US,en;q=0.9",
+        connection = "keep-alive",
+        host = "127.0.0.1:8600",
+        `upgrade-insecure-requests` = "1",
+        `user-agent` = "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/68.0.3440.75 Chrome/68.0.3440.75 Safari/537.36"
+      ),
+      rook.url_scheme = "http",
+      HTTP_ACCEPT_ENCODING = "gzip, deflate, br",
+      REQUEST_METHOD = "GET",
+      HTTP_USER_AGENT = "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/68.0.3440.75 Chrome/68.0.3440.75 Safari/537.36",
+      HTTP_HOST = "127.0.0.1:8600"
+    )
+  to_remove <-
+    shinyloadtest:::headers_to_remove(rook_req$HTTP_CONNECTION)
```

Comment:
Shouldn't use `:::` here - testthat automatically uses `load_all` so even internal functions will be available.